ReadmeSrc/Rows.js

```diff
@@ -0,0 +1,65 @@
+const headers = [
+  "Package",
+  //   'Version',
+  "Automatic Vendor Prefixing",
+  "Pseudo Classes",
+  "Media Queries",
+  "Styles As Object Literals",
+  "Extract CSS File",
+  "Package Stats"
+];
+
+const link = obj => `[${obj.text}](${obj.href})`;
+const links = objs => objs.map(link).join(" + ");
+const version = obj => obj;
+const symbol = obj => (obj === true ? "✓" : "");
+
+const stats = obj => {
+  return obj
+    .map((stat, index) => {
+      //prettier-ignore
+      const spacer = index > 0 ? '<div>____________</div>': ''
+
+      //prettier-ignore
+      const openIssues =`<div><sub><img src="./ReadmeSrc/img/hand-paper-o.svg" width="12" />  ${stat.githubData.open_issues}</sub></div>`
```

Comment:
```js
const divWithIcon = (icon, value) => 
  `<div><sub><img src="./ReadmeSrc/img/${icon}.svg" width="12" />${value}</sub></div>`
const openIssues = divWithIcon("hand-paper-o", stat.githubData.open_issues);
```

Also not sure why you want to use `//prettier-ignore`. Formatting here does not contain any value. Idea of prettier is to stop worrying about formatting. And you need `//prettier-ignore` only when, formatting itself is valuable.

Reply:
I didn't like the way prettier formatted that part but ok, I'll remove the ignore comments.

![image](https://user-images.githubusercontent.com/37283/39299312-56c6bb7c-4940-11e8-8633-1a4ab360b4fc.png)

Reply:
> I didn't like

This is the idea. You just need to stop care. I do not like semicolons either, but I'm using prettier without config (e.g. with semicolons). And I stopped to care what is happening with formatting, because I do not need to touch it. It is not my job. This is the part of the code which I will never need to touch.

Reply:
You are right. 
I removed them

Reply:
```js
const divWithIcon = (icon, value) => 
  `<div><sub><img src="./ReadmeSrc/img/${icon}.svg" width="12" />${value}</sub></div>`
const openIssues = divWithIcon("hand-paper-o", stat.githubData.open_issues);
```